### PR TITLE
Add `wp plugin language uninstall --all`

### LINKED ISSUE #122: Add `--all` flag to `uninstall` commands
## Feature Request

- [x] Yes, I reviewed the [contribution guidelines](https://make.wordpress.org/cli/handbook/contributing/).

**Describe your use case and the problem you are facing**

The `plugin install` and `theme install` commands support `--all` to quickly install languages for all plugins/themes.

The `uninstall` commands don't support it.

**Describe the solution you'd like**

Adding this flag to the `plugin install` and `theme install` commands would make them more consistent.

### PR DESCRIPTION
**Pull request by swissspidy**
Contains follow-up fixes from #124 as well.

Fixes #122.

### CODE REVIEW

File: src/Plugin_Language_Command.php [hunk 1]
```diff
@@ -400,24 +439,87 @@ public function uninstall( $args, $assoc_args ) {
...
+				if ( count( $files_to_remove ) === $count_files_removed ) {
+					$result['status'] = 'uninstalled';
+					++$successes;
+					\WP_CLI::log( "Language '{$language_code}' for '{$plugin}' uninstalled." );
+				} elseif ( $count_files_removed ) {
+					\WP_CLI::log( "Language '{$language_code}' for '{$plugin}' partially uninstalled." );
+					$result['status'] = 'partial uninstall';
+					++$errors;
+				} elseif ( $had_one_file ) { /* $count_files_removed == 0 */
+						\WP_CLI::log( "Couldn't uninstall language '{$language_code}' from theme {$plugin}." );
```
Comment (danielbachhuber) [inline src/Plugin_Language_Command.php:505]:
```suggestion
						\WP_CLI::log( "Couldn't uninstall language '{$language_code}' from plugin {$plugin}." );
```

File: src/Plugin_Language_Command.php [hunk 2]
```diff
@@ -400,24 +439,87 @@ public function uninstall( $args, $assoc_args ) {
...
+
+				$count_files_removed = 0;
+				$had_one_file        = 0;
+				foreach ( $files_to_remove as $file ) {
+					if ( $wp_filesystem->exists( $file ) ) {
+						$had_one_file = 1;
+						if ( $wp_filesystem->delete( $file ) ) {
+							++$count_files_removed;
+						} else {
+							\WP_CLI::error( "Couldn't uninstall language: $language_code from theme $plugin." );
```
Comment (danielbachhuber) [inline src/Plugin_Language_Command.php:491]:
```suggestion
							\WP_CLI::error( "Couldn't uninstall language: $language_code from plugin $plugin." );
```

**Review by danielbachhuber — CHANGES_REQUESTED** [review]

**Comment by danielbachhuber** [comment]
@swissspidy Does the test failure make sense to you?

**Comment by swissspidy** [comment]
> @swissspidy Does the test failure make sense to you?

For some reason the order of the plugins in the plugin list is different 🤷  Just added some hardening to the test to guard against this.

**Review by danielbachhuber — APPROVED** [review]

**Pull request merged**